ironfish/src/rpc/routes/wallet/types.ts

```diff
@@ -142,11 +149,7 @@ export type RpcAccountImport = {
   publicAddress: string
   spendingKey: string | null
   createdAt: { hash: string; sequence: number } | null
-  multiSigKeys?: {
-    identifier: string
-    keyPackage: string
-    proofGenerationKey: string
-  }
+  multiSigKeys: RpcMultiSigKeys | null
```

Comment:
Should this be `multiSigKeys?: RpcMultiSigKeys | null`

Reply:
Updated